.gitignore

```diff
@@ -12,3 +12,7 @@
 hs_err_pid*
 DataManager/src/Tests/
 Serverp/client_log.dat
+Serverp/version.dat
+/nbproject/private/
+/AndroidAppTests/nbproject/private/
```

Comment:
Well as long as this solves the merge conflict I am happy. The folders may need a rename though.